docs/blog/_posts/2021-04-21-scala3-rc3.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-page
+title: Scala 3.0.0-RC3 – bug fixes for 3.0.0 stable
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2021-04-21
+---
+
+Hello! With this blog article, we would like to announce the release of Scala 3.0.0-RC3. Some critical bugs were discovered during the previous release cycle the fixes to which had to be included in 3.0.0 stable. Therefore, we are having RC3 to give the community time to test our fixes.
```

Comment:
```suggestion
Hello! With this blog article, we would like to announce the release of Scala 3.0.0-RC3. Some critical bugs were discovered during the previous release cycle, whose fixes had to be included in 3.0.0 stable. Therefore, we are having RC3 to give the community time to test our fixes.
```

Reply:
Dare to be colloquial! I think "had to be included" is the problem. "Some bugs were found which had to be fixed." "Some bugs were found, so a release was needed." etc. "We discovered some bugs and wanted to include them in 3.0." I was tempted to read some wikipedia for the grammar terms but instead I stepped out of the rabbit hole. (I read as far as "ergative" and "labile".) It's just a modal but has the force of passive, or lack of force, especially following "were discovered" in passive voice. Also, "we're having a party" suggests that RC3 is a party. We are releasing RC3, or we decided on an RC3 release in order to... etc. I hope that if someone on the team starts a side gig as a DJ, they go by "RC3". Because it means party. Thanks for TIL ergative.